### PR TITLE
Fixed support service issue

### PR DESCRIPTION
**Pull request by KKarlash**
@bonewell 
@VladSemenyuk 
@DKlimenko 
@giangtqh 
@thinhts

### CODE REVIEW

File: src/components/vr_cooperation/src/command/support_service_request.cc [hunk 1]
```diff
@@ -53,6 +53,7 @@ void SupportServiceRequest::Execute() {
   message_.set_rpc(vr_hmi_api::SUPPORT_SERVICE);
   message_.set_rpc_type(vr_hmi_api::REQUEST);
   message_.set_correlation_id(parent()->GetNextCorrelationID());
+  parent()->RegisterRequest(message_.correlation_id(), this);
   parent()->SendMessageToHMI(message_);
```
Comment (giangtqh) [inline src/components/vr_cooperation/src/command/support_service_request.cc]: Forgot to add_observer(), You can add_observer in SendMessageToHMI of BaseGpbRequest and use base method here.

**Comment by VladSemenyuk** [comment]
Reviewed

File: src/components/vr_cooperation/src/vr_module.cc [hunk 2]
```diff
@@ -301,6 +305,12 @@ void VRModule::SendUnsupportedServiceResponse(
   service()->SendMessageToMobile(msg);
 }
 
+void VRModule::RegisterRequest(int32_t correlation_id,
+                               commands::Command* command) {
+  LOG4CXX_AUTO_TRACE(logger_);
+  request_controller_.AddRequest(correlation_id, command);
```
Comment (DKlimenko) [inline src/components/vr_cooperation/src/vr_module.cc]: may be we need to check here if service is supported by HMI?

Comment (KKarlash) [inline src/components/vr_cooperation/src/vr_module.cc]: It's used before sending support service request

**Comment by giangtqh** [comment]
reviewed

**Pull request merged**